packages/react/src/components/radio-button/radio-button.tsx

```diff
@@ -0,0 +1,130 @@
+import { ChangeEvent, FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { focus } from '../../utils/css-state';
+import { VisuallyHidden } from '../visually-hidden/visuallyhidden';
+import { useId } from '../../hooks/use-id';
+
+const StyledInput = styled.input<{ disabled?: boolean }>`
+    appearance: none;
+    background-color: ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-background-color'] : theme.component['radio-button-background-color'])};
+    border: 1px solid ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-border-color'] : theme.component['radio-button-border-color'])};
+    border-radius: 50%;
+    color: ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-background-color'] : theme.component['radio-button-background-color'])};
+    display: inline-block;
+    height: var(--size-1x);
+    margin: 0;
+    position: relative;
+    width: var(--size-1x);
+    
+    ${(theme) => focus(theme, { selector: '+' })}
+     
+    &:checked {
+        border: 2px solid ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-border-color'] : theme.component['radio-button-checked-border-color'])};
+        &::after {
+            background-color: ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-border-color'] : theme.component['radio-button-checked-background-color'])};
+            border-radius: 50%;
+            content: '';
+            display: block;
+            height: var(--size-half);
+            left: 50%;
+            position: absolute;
+            top: 50%;
+            transform: translate(-50%, -50%);
+            width: var(--size-half);
+        }
+    }
+        
+    &:hover [type='radio'] {
+        border: 1px solid ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-hover-border-color'] : theme.component['radio-button-hover-border-color'])};
+    }
+`;
+
+interface StyledLabelProps {
+    disabled?: boolean;
+}
+
+const StyledLabel = styled.label<StyledLabelProps>`
```

Comment:
Enlever `disabled` de l'interface aussi, ca revient a enlever l'interface completement

```suggestion
const StyledLabel = styled.label`
```